examples/tutorials/api/estimators.py

```diff
@@ -0,0 +1,404 @@
+#
+
+energy_edges = np.geomspace(0.7, 100, 9) * u.TeV
+
+fp_estimator = FluxPointsEstimator(
+    source="crab",
+    energy_edges=energy_edges,
+)
+
+# %%time
+fp_result = fp_estimator.run(datasets=datasets)
+
+######################################################################
+# Accessing and visualising the results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+print(fp_result)
+
+######################################################################
+# We can specify the SED type to plot:
+#
+fp_result.plot(sed_type="dnde")
+plt.show()
+
+######################################################################
+# Here we show how you can plot a different plot type and define the axes units.
+
+ax = plt.subplot()
+ax.xaxis.set_units(u.eV)
+ax.yaxis.set_units(u.Unit("TeV cm-2 s-1"))
+fp_result.plot(ax=ax, sed_type="e2dnde")
+plt.show()
+
+######################################################################
+# The actual data members are N-dimensional `~gammapy.maps.region.ndmap.RegionNDMap` objects. So you can
+# also plot them:
+
+print(type(fp_result.dnde))
+
+######################################################################
+#
+fp_result.dnde.plot()
+plt.show()
+
+######################################################################
+# Access the data:
+
+print(fp_result.e2dnde.quantity.to("TeV cm-2 s-1"))
+
+######################################################################
+#
+print(fp_result.dnde.quantity.shape)
+
+######################################################################
+#
+print(fp_result.dnde.quantity[:, 0, 0])
+
+######################################################################
+# Or even extract an energy range:
+
+fp_result.dnde.slice_by_idx({"energy": slice(3, 10)})
+
+
+######################################################################
+# A note on the internal representation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The result contains a reference spectral model, which defines the spectral shape.
+# Typically, it is the best fit model:
+
+print(fp_result.reference_model)
+
+######################################################################
+# `FluxPoints` are the represented by the "norm" scaling factor with
+# respect to the reference model:
+
+fp_result.norm.plot()
+plt.show()
+
+######################################################################
+# Dataset specific quantities ("counts like")
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# While the flux estimate and associated errors are common to all datasets,
+# the result also stores some dataset specific quantities, which can be useful
+# for debugging. The `~gammapy.maps.region.ndmap.RegionNDMap` allows for plotting of multidimensional data
+# as well, by specifying the primary `axis_name`:
+
+
+fp_result.counts.plot(axis_name="energy")
+plt.show()
+
+######################################################################
+#
+fp_result.npred.plot(axis_name="energy")
```

Comment:
Maybe you can remind what is npred, npred_excess... as reminder in this tutorial